src/02-data-types.js

```diff
@@ -127,7 +163,13 @@ function isOdd(num) {}
  *  isTruthy(null);
  *  //> false
  */
-function isTruthy(val) {}
+function isTruthy(val) {
+  if (val) {
```

Comment:
Remember we can use the !! operator here to check the if a value is truthy and simply return it, i.e. return !!val